R/Module.R

```diff
@@ -131,6 +131,8 @@ new_dummyObject <- function(...)                                # #nocov
 
 # class method for $initialize
 cpp_object_initializer <- function(.self, .refClassDef, ..., .object_pointer){
+    # force finalize method to be materialized
+    invisible(.self$finalize)
```

Comment:
Wow. A one-liner to fix a woe.  I presume you played with some Rprintf or REprintf here to see it hits?